src/application/services/useNoteSettings.ts

```diff
@@ -17,15 +17,15 @@ interface UseNoteSettingsComposableState {
    *
    * @param id - note id
    */
-  load: (id: NoteId) => Promise<void>;
+  loadSettings: (id: NoteId) => Promise<void>;
```

Comment:
I guess you renamed functions to make it clear what they do in the place where they are called? No need to rename the functions themselves for that, you can do it when destructuring return value of `useNoteSettings`

```
const { load: loadSettings } = useNoteSettings();
```